tests/voq/test_voq_nbr.py

```diff
@@ -32,6 +37,134 @@
 NEW_MAC = "00:01:94:00:00:01"
 
 
+def check_bgp_restored(duthosts, all_cfg_facts):
+    """
+    Checks for bgp neighbors to be established.
+
+    Args:
+        duthosts: duthosts fixture
+        all_cfg_facts: fixture from voq/conftest.py
+
+    Returns:
+        True if all neighbors are established, False if not.
+    """
+    down_nbrs = 0
+    for node in duthosts.frontend_nodes:
+        for asic in node.asics:
+            asic_cfg_facts = all_cfg_facts[node.hostname][asic.asic_index]['ansible_facts']
+            if 'BGP_NEIGHBOR' not in asic_cfg_facts:
+                continue
+
+            if node.is_multi_asic:
```

Comment:
we can use` asic.bgp_facts()`. This will handle single and multi asic

Reply:
Fixed in latest.